main.go

```diff
@@ -707,6 +708,7 @@ func mustPrepareService(generateAutoKeyFile bool) (*service.Service, service.Boo
 		ArangodPath:             arangodPath,
 		ArangoSyncPath:          arangoSyncPath,
 		ArangodJSPath:           arangodJSPath,
+		AdvertisedEndpoint:      advertisedEndpoint,
```

Comment:
IMO the advertised endpoint should be checked (when non-empty) to make sure it is a valid format.

Reply:
Again, also here we had decided to not do any sanity checks. It it does not suffice the endpoint specification, the affected services will complain. Effectively like wrong passthrough options

Reply:
Not so sure on the "no sanity checks". If we decided to that, it sounds like a bad decision.

Reply:
Now test was added. Problem solved.